sentry/src/main/java/io/sentry/util/PropagationTargetsUtils.java

```diff
@@ -1,14 +1,13 @@
-package io.sentry;
+package io.sentry.util;
 
 import java.net.URI;
 import java.util.List;
+import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 
-/**
- * Checks if an URL matches the list of origins to which `sentry-trace` header should be sent in
- * HTTP integrations.
- */
-public final class TracePropagationTargets {
+/** Checks if an URL matches the list of origins. */
+@ApiStatus.Internal
```

Comment:
Since this is reused in different features, I've renamed to make it more generic.

Reply:
Please make sure we don't loose the changes regarding not throwing on a broken regex that are on `main`.